contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import psutil
+
+from opencensus.metrics.export.gauge import DerivedLongGauge
+from opencensus.metrics.export.standard_metrics import BaseStandardMetric
+from opencensus.metrics.export.standard_metrics import StandardMetricsProducer
+
+
+class StandardMetricsType(object):
+    AVAILABLE_MEMORY = "\\Memory\\Available Bytes"
+
+
+# Definitions taken from psutil docs
+# https://psutil.readthedocs.io/en/latest/
+class AvailableMemoryStandardMetric(BaseStandardMetric):
```

Comment:
Like `StandardMetricsProducer` below, since the only state in this class in the registry -- and I don't think you need the registry here -- I don't see a reason to have this class.

Flattening this class could look like:

```python
def get_available_memory():
    return psutil.virtual_memory().available


def get_available_memory_metric():
    gauge = DerivedLongGauge(
        AVAILABLE_MEMORY,
        'Amount of available memory in bytes',
        'byte',
        [])
    gauge.create_default_time_series(get_available_memory)


class AzureStandardMetricsProducer(MetricProducer):
    def __init__(self):
        self.registry = Registry()
        self.registry.add_gauge(get_available_memory_metric())

    def get_metrics(self):
        return self.registry.get_metrics()


producer = AzureStandardMetricsProducer()
```

This is just one option -- it'd be fine to have separate classes for each metric still instead -- but I don't see the point of passing the registry to the metric.